### DISCUSSION TITLE
Azure pipeline - build .ipa

### DISCUSSION THREAD

**Discussion by hamada-applicita** [unoplatform/uno#14438] [Q&A]
I need to build an iOS ipa file from Azure pipelines on an MS hosted mas build agent.

Is there any documentation or code samples on how to do this.

**Comment by hamada-applicita** [comment]
I found the info now - https://github.com/nventive/UnoApplicationTemplate/blob/main/build/steps-build-ios.yml

**Reply by jeromelaban** [reply]
You can also use the VS Wizard to [include CI templates](https://platform.uno/docs/articles/getting-started/wizard/using-wizard.html#11-ci-pipeline).
